src/components/stories/Card.js

```diff
@@ -37,6 +37,6 @@ storiesOf('Card', module)
       <img src="http://www.top13.net/wp-content/uploads/2015/10/perfectly-timed-funny-cat-pictures-5.jpg" />
       <h1>Now with footer.</h1>
       <p>header and footer elements are also not subject to margin</p>
-      <footer style={styles.footer}>&copy; Thousand Cats Corp</footer>
+      <footer style={styles("footer")}>&copy; Thousand Cats Corp</footer>
```

Comment:
should this be a class in place of style?

Reply:
Actually, this should stay the way it was before, and it got changed to this when changing the prefixer to a function. In this case the `styles` object is a manually constructed object earlier in the file, and has nothing to do with the prefixer or CSS modules. I'll change this back.